core/templates/dev/head/domain/collection/CollectionUpdateService.js

```diff
@@ -99,6 +99,28 @@ oppia.factory('CollectionUpdateService', [
         return _getParameterFromChangeDict(changeDict, 'exploration_id');
       };
 
+      // Validates the tags for the collection
+      var _validateTags = function(tags) {
+        // Check to ensure that all tags follow the format specified in
+        // TAG_REGEX.
+        for (var i = 0; i < tags.length; i++) {
```

Comment:
A nicer way to do this might be to use javascript's `some()` function (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)